UEBPrep/config/initializers/devise.rb

```diff
@@ -31,7 +31,7 @@
   # session. If you need permissions, you should implement that in a before filter.
   # You can also supply a hash where the value is a boolean determining whether
   # or not authentication should be aborted when the value is not present.
-  # config.authentication_keys = [:email]
+  config.authentication_keys = [:email, :username]
```

Comment:
pretty sure we can update this to username only. or username and password actually. Going to try it out.